jsk_fetch_robot/fetcheus/fetch-interface.l

```diff
@@ -191,7 +198,38 @@ Example: (send self :gripper :position) => 0.00"
     (send-super* :speak-en text :topic-name topic-name args))
   (:speak-jp (text &rest args &key (topic-name "robotsound_jp") &allow-other-keys)
     (send-super* :speak-jp text :topic-name topic-name args))
-  )
+  (:add-headbox-collision-object ()
+    (let ()
+      ;; fetch が reset-pose をとっていることが前提
+      (setq *fetch-headbox* (make-cube 100 201 120))
+      (send *fetch-headbox* :move-coords (send robot :head_pan_link_lk :worldcoords)
+                                         (send robot :base_link_lk :worldcoords))
+      (send *fetch-headbox* :translate #f(33.75 0 150) (send robot :head_pan_link_lk :worldcoords))
+      (send *co* :add-attached-object *fetch-headbox* "head_pan_link"
+                 :frame_id "head_pan_link"
+                 :object_id "fetchheadbox")))
+  (:delete-headbox-collision-object ()
+    (send *co* :delete-attached-object-by-id "fetchheadbox"))
+  (:add-keepout-collision-object ()
+    (let ((cube (make-cube 200 350 50))
+          (keepout (make-cylinder 300 50)))
+      (send cube :translate #f(-120 0 0))
+      (send keepout :translate #f(0 0 -25))
+      (setq keepout (body- keepout cube))
+      (send keepout :translate #f(0 0 375))
+      (send *co* :add-object keepout :frame-id base-frame-id :object-id "keepout")))
+  (:delete-keepout-collision-object ()
+    (send *co* :delete-object-by-id "keepout"))
+  (:add-ground-collision-object ()
+    (let ((cube (make-cube 200 350 50))
+          (ground (make-cylinder 1000 50)))
+      (send cube :translate #f(-120 0 0))
+      (send ground :translate #f(0 0 -25))
+      (setq ground (body- ground cube))
+      (send ground :translate #f(0 0 -25))
+      (send *co* :add-object ground :frame-id base-frame-id :object-id "ground")))
```

Comment:
if https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/423/files is correct, this commit is wrong.

Reply:
this is correct because this pr uses `:frame-id`, not`:frame_id`.